css/excercise_one/part-d/scss/excercise_one.scss

```diff
@@ -0,0 +1,405 @@
+.top-rounded { border-top-left-radius: 20px; border-top-right-radius: 20px;}
+.letter-spacing-negative { letter-spacing: -1px; }
+
+.gap-xs { gap: map-get(map-get($gap, lg), xsmall) }
+.gap-sm { gap: map-get(map-get($gap, lg), small) }
+.gap-md { gap: map-get(map-get($gap, lg), medium) }
+.gap-lg { gap: map-get(map-get($gap, lg), large) }
+.gap-xl { gap: map-get(map-get($gap, lg), xlarge) }
+
+.dark-blue-gradient { background: linear-gradient(46.99deg, #0F58B7 -4.4%, #62D9FF 120.42%); }
+.light-blue-gradient { background: linear-gradient(46.77deg, #0F99B7 -0.79%, #69ECDC 131.52%); }
+.background-clip-text { -webkit-background-clip: text; }
+.text-transparent { color: transparent; }
+
+.div-gradient {
+  background-image: linear-gradient(46.99deg, #0F58B7 -4.4%, #62D9FF 120.42%);
+  display: inline-block;
+  -webkit-background-clip: text;
+}
+
+.font-white { color: $light; }
+.background-white { background-color: $light; z-index: 1;}
+
+.opacity-2 { opacity: 2; }
+.opacity-1 { opacity: 0.2;}
+
+.standard-border { border: 1px solid #1C1C1C; }
+.border-thin { border: 0.9px solid rgba(0, 0, 0, 0.03); }
+
+.w-xs { width: map-get(map-get($width, lg), xsmall) }
+.w-sm { width: map-get(map-get($width, lg), small) }
+.w-md { width: map-get(map-get($width, lg), medium) }
+.w-lg { width: map-get(map-get($width, lg), large) }
+.w-xl { width: map-get(map-get($width, lg), xlarge) }
+.w-xxl { width: map-get(map-get($width, lg), xxlarge) }
+.w-xxxl { width: map-get(map-get($width, lg), xxxlarge) }
+.w-60 { width: map-get(map-get($width, lg), 60) }
+.w-100 { width: map-get(map-get($width, lg), 100) }
+
+.h-xs { height: map-get(map-get($height, lg), xsmall) }
+.h-sm { height: map-get(map-get($height, lg), small) }
+.h-md { height: map-get(map-get($height, lg), medium) }
+.h-lg { height: map-get(map-get($height, lg), large) }
+.h-xl { height: map-get(map-get($height, lg), xlarge) }
+.h-xxl { height: map-get(map-get($height, lg), xxlarge) }
+.h-xxxl { height: map-get(map-get($height, lg), xxxlarge) }
+.h-100 { height: map-get(map-get($height, lg), 100) }
+
+.border-transparent { border-width: 0; }
+.standard-shadow { box-shadow: 2px 2px 10px 0 rgba(0, 0, 0, 0.25); }
+
+.primary-background-color { background-color: $light;}
+.secondary-background { background-color: $secondary-color}
+.tertiary-background-color {background-color: $primary-color}
+.secondary-color { color: $primary-color;}
+.tertiary-color { color: $primary-color-light;}
+.non-wrap { white-space: nowrap; }
+
+.object-fit-contain {
+  object-fit: contain;
+}
+
+.custom-list {
+  list-style-type: none;
+  padding-left: 0;
+  > li {
+    background-image: url('../../../../img/tic.png');
+    background-position: 0 5px;
+    background-repeat: no-repeat;
+    margin: 5px 0;
+    padding-left: 30px;
+  }
+}
+
+.custom-input{
+  background: transparent;
+  border: none;
+  border-bottom: 1px solid $light;
+  color: $light;
+  margin: 0;
+  outline: none;
+  padding: 0;
+  width: 100%;
+}
```

Comment:
alot of this can be done with different classes that you got, maybe try doing it and just leave the once you don't have here